test/extended/quota/resourcequota.go

```diff
@@ -216,6 +217,75 @@ var _ = g.Describe("[sig-api-machinery][Feature:ResourceQuota]", func() {
 			})
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("when exceed openshift.io/image-tags will ban to create new image references in the project [Skipped:Disconnected]", func() {
+			testProject := oc.SetupProject()
+			testResourceQuotaName := "my-image-tag-quota-" + testProject
+			clusterAdminKubeClient := oc.AdminKubeClient()
+
+			lr := &corev1.LimitRange{
+				ObjectMeta: metav1.ObjectMeta{Name: testResourceQuotaName, Namespace: testProject},
+				Spec: corev1.LimitRangeSpec{
+					Limits: []corev1.LimitRangeItem{
+						{
+							Type: "openshift.io/ImageStream",
+							Max: corev1.ResourceList{
+								"openshift.io/image-tags": resource.MustParse("2"),
+							},
+						},
+					},
+				},
+			}
+
+			g.By("create the image-tags and checking the usage")
+			_, err := clusterAdminKubeClient.CoreV1().LimitRanges(testProject).Create(context.Background(), lr, metav1.CreateOptions{})
```

Comment:
Do we consider removing the resource we created in test after test finished?

Reply:
@wangke19 No need it created with temp namespace and it get deleted when case completed.
```
Dec 19 17:29:43.438: INFO: Deleted {user.openshift.io/v1, Resource=users e2e-test-object-count-rq-lmwkb-user}, err: <nil>
 Dec 19 17:29:43.685: INFO: Deleted {oauth.openshift.io/v1, Resource=oauthclients e2e-client-e2e-test-object-count-rq-lmwkb}, err: <nil>
 Dec 19 17:29:43.937: INFO: Deleted {oauth.openshift.io/v1, Resource=oauthaccesstokens sha256~dVTBrllwH1oEHTvbEntb0T7iHpUoBdkbtbjamqQDEBA}, err: <nil>
 Dec 19 17:29:44.187: INFO: Deleted {user.openshift.io/v1, Resource=users e2e-test-object-count-rq-pxwxp-user}, err: <nil>
 Dec 19 17:29:44.444: INFO: Deleted {oauth.openshift.io/v1, Resource=oauthclients e2e-client-e2e-test-object-count-rq-pxwxp}, err: <nil>
 Dec 19 17:29:44.745: INFO: Deleted {oauth.openshift.io/v1, Resource=oauthaccesstokens sha256~dqfyaEX6VqZ36dEfm95xvLK0Kr9zSFlrmC1-9dqhOLc}, err: <nil>
  STEP: Destroying namespace "e2e-test-object-count-rq-lmwkb" for this suite. @ 12/19/23 17:29:44.746
  STEP: Destroying namespace "e2e-test-object-count-rq-pxwxp" for this suite. @ 12/19/23 17:29:44.994
 • [24.546 seconds]
 ------------------------------

 Ran 1 of 1 Specs in 24.546 seconds
 SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
rahulgangwar@rgangwar-mac origin % oc get limits -A     
No resources found
```